src/Makefile

```diff
@@ -386,6 +386,13 @@ ifeq ($(avx2),yes)
 	endif
 endif
 
+ifeq ($(sse41),yes)
+	CXXFLAGS += -DUSE_SSE41
+	ifeq ($(comp),$(filter $(comp),gcc clang mingw msys2))
+		CXXFLAGS += -msse4
```

Comment:
Is "-msse4.1" correct?  Could you please double check it?

Reply:
-msse4 should be fine. -msse41 gives error

> g++: error: unrecognized command line option ‘-msse41’; did you mean ‘-msse4’?

Reply:
Oops, missed the dot. Must be -msse4.1 or  -msse4.2, of course!
From here: https://stackoverflow.com/questions/10686638/whats-the-difference-among-cflgs-sse-options-of-msse-msse2-mssse3-msse4

> What's the point of msse4? As far as I can tell it is the same as msse4.2.

Reply:
> What's the point of msse4? As far as I can tell it is the same as msse4.2.
I guess that msse4 and msse4.1 are equivalent.  In msse4.2, additional extended instructions for string processing and CRC are available.

Reply:
Done! Thank you.